parsers/nmparser/parse_run_details.go

```diff
@@ -71,5 +60,14 @@ func ParseRunDetails(lines []string) RunDetails {
 		covarianceTime,
 		functionEvaluations,
 		significantDigits,
+		// TODO: populate with values
+		"",  // ProblemText 		string
+		"",  // ModFile             string
+		nil, // EstimationMethod    []string
```

Comment:
should probably be an empty []string not nil right?

Reply:
ok, changed